SignalServiceKit/protobuf/OWSSignalServiceProtos.proto

```diff
@@ -37,35 +37,18 @@ message Content {
   optional NullMessage nullMessage = 4;
 }
 
```

Comment:
Most of the changes in the proto files were in an effort to minimize the diff with upstream (signal-android)